Woof/WoofTests/Model/LocalStorage/LocalStorageDeleteTests.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+
+final class LocalStorageDeleteTests: XCTestCase {
+    private var localStorage: LocalStorage!
+    private let defaultValueToSave = "Hello, World!".data(using: .utf8)!
```

Comment:
nit: 

You can make a static constant within a separate enum to reuse the `defaultValueToSave` across test cases and follow DRY.

Reply:
fixed